pkg/scheduler/internal/parallelize/parallelism.go

```diff
@@ -21,15 +21,15 @@ import (
 	"math"
 
 	"k8s.io/client-go/util/workqueue"
+	"k8s.io/kubernetes/pkg/scheduler/apis/config"
 )
 
-const parallelism = 16
-
 // chunkSizeFor returns a chunk size for the given number of items to use for
 // parallel work. The size aims to produce good CPU utilization.
-func chunkSizeFor(n int) workqueue.Options {
+func chunkSizeFor(n int, in *config.KubeSchedulerConfiguration) workqueue.Options {
```

Comment:
receive the int instead. There's no really a reason for this package to depend on configuration for one value.

Reply:
Alternatively, you could expose a function `SetParallelism` that is called once once the configuration has been loaded.